### ISSUE TITLE
Script fails to multiplex unique file names

### ISSUE THREAD

**Issue by leptoid** [seiya-dev/funimation-downloader-nx#5]
The show "Seven Mortal Sins" has an episode labeled 4.5, and where the script will download it, the script fails to multiplex the files into a single file.  I assume it is because the script or software is having an issue with the ithe irregular episode number.

`
`"node" "%~dp0.\scripts\funidl.js" -s 369617 --sel 4.5`
`

**Comment by seiya-git** [comment]
Use latest version:
```funi -s 369617 --sel S001```

**Comment by leptoid** [comment]
My issue looks more like this: 
![untitled](https://user-images.githubusercontent.com/18079714/27264333-cff499dc-5441-11e7-9590-73d0837aceda.png)

This leaves the following files in output directory:
[Funimation] Seven Mortal Sins - 04.5 [720p].meta
[Funimation] Seven Mortal Sins - 04.5 [720p].ts
[Funimation] Seven Mortal Sins - 04.track_256.264
[Funimation] Seven Mortal Sins - 04.track_257.aac

**Comment by leptoid** [comment]
Obviously the error stems "[Funimation] Seven Mortal Sins - 04.track_256.264" having an incorrect name.
It should be named "**[Funimation] Seven Mortal Sins - 04.5 [720p].track_256.264**"
This is the same for "[Funimation] Seven Mortal Sins - 04.track_257.aac", as it should be "[Funimation] Seven Mortal Sins - 04.5 [720p].track_257.aac"

By manually renaming the files, rerunning the script, telling it not to overwrite files, it was able to multiplex the files correctly.

Basically, the code for naming files may have a flaw regarding files with "." in their name when handling audio files.  This may be an issue with one of the script dependencies as opposed to this script in itself.

**Comment by seiya-git** [comment]
Ok. I'll look into it.

**Comment by seiya-git** [comment]
Fixed with latest commit.

**Comment by leptoid** [comment]
Works for me now.  Had to also update npm to version 5+
